src/index.tsx

```diff
@@ -72,14 +82,13 @@ interface State {
   lastDrag: Drag | null;
   heights: { [key: string]: HeightData } | null;
 }
-export default class DraggableList<
+export class DraggableList<
   I,
   C,
-  T extends React.Component<Partial<TemplateProps<I, C>>>
+  T extends React.Component<Partial<TemplateProps<I, C, T>>>
 > extends React.Component<Props<I, C, T>, State> {
   public static propTypes = {
     itemKey: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
-    template: PropTypes.func,
```

Comment:
Let's get `renderTemplate: PropTypes.func` into the propTypes here.

Reply:
I considered dropping `prop-types` in their entirety. Would you be opposed?